stripper/ze_diverted_destiny_c2f3.cfg

```diff
@@ -0,0 +1,381 @@
+;stripper by koen (STEAM_1:1:114921174)
+
+;remove broken trigger_hurt for solar flare
+filter:
+{
+	"classname" "trigger_hurt"
+	"targetname" "solarflare_item_h"
+	"hammerid" "328010"
+}
+
+;adjust solar flare damage to prevent repeat killer
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "solarflare_item_h"
+		"hammerid" "328019"
+	}
+	replace:
+	{
+		"damage" "2500"
+	}
+}
+
+;fix teleport avoidance spot at ball dodging platform
+add:
+{
+	"classname" "trigger_teleport"
+	"targetname" "tp12"
+	"target" "tp12_des"
+	"spawnflags" "1"
+	"StartDisabled" "1"
+	"origin" "-9131 6385 10144"
+	"model" "*169"
+}
+
+;fix zombies getting stuck in the floor after boss fight teleport
+modify:
+{
+	match:
+	{
+		"classname" "info_teleport_destination"
+		"origin" "-9648 5536 5136"
+	}
+	replace:
+	{
+		"origin" "-9648 5536 5196"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "info_teleport_destination"
+		"origin" "-8576 6560 5136"
+	}
+	replace:
+	{
+		"origin" "-8576 6560 5196"
+	}
+}
+
+;prevent people from getting stuck in a tree
+add:
+{
+	"classname" "trigger_multiple"
+	"targetname" "treestuck"
+	"StartDisabled" "0"
+	"spawnflags" "4097"
+	"origin" "-644 7621 384"
+	"model" "*78"
+	"rendermode" "10"
+	"OnStartTouch" "!activatorAddOutputorigin -652 7500 3720-1"
+}
```

Comment:
Nit, but wouldn't it be more graceful to have this be a func_brush/wall_toggle "clip"?